examples/introduction.ipynb

```diff
@@ -23,7 +23,56 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "sc = Sidecar(title='Sidecar Output')\n",
+    "from bokeh.io import push_notebook, show, output_notebook\n",
```

Comment:
Let's make the examples use standard widgets so everyone can run them with just ipywidgets installed.